src/olympia/devhub/tasks.py

```diff
@@ -143,17 +145,21 @@ def wrapper(task, id_, hash_, *args, **kw):
 
 
 @validation_task
-def validate_file_path(path, hash_, listed, **kw):
+def validate_file_path(path, hash_, listed, is_webextension=False, **kw):
     """Run the validator against a file at the given path, and return the
     results.
 
     Should only be called directly by ValidationAnnotator."""
-
+    run_linter = (
```

Comment:
I wouldn't make a variable here (and below), to avoid the waffle db/cache queries when we are not dealing with a webextension.